behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/aksjonspunkt/AksjonspunktKodeDefinisjon.java

```diff
@@ -58,6 +58,7 @@ public class AksjonspunktKodeDefinisjon {
     public static final String OVERSTYRING_AV_BEREGNINGSAKTIVITETER_KODE = "6014";
     public static final String OVERSTYRING_AV_BEREGNINGSGRUNNLAG_KODE = "6015";
     public static final String OVERSTYRING_AV_DEKNINGSGRAD_KODE = "6016";
+    public static final String OVERSTYRING_AV_FAKTA_OM_FØDSEL_KODE = "6019";
```

Comment:
Sikre på at dere vil ha en overstyring for alle datoer, vs å skille overstyring av termindato og overstyr fødselsdato? Det er ganske ulike use-cases - termindato ifm permatur og føsdselsdato ifm utland.
Se ellers ForvaltningSøknadRestTjeneste.